controllers/apply/form-router/lib/hotjar.js

```diff
@@ -4,7 +4,7 @@ function getShortId(formId) {
     let hotjarAlias;
     switch (formId) {
         case 'awards-for-all':
-            hotjarAlias = 'AFA';
+            hotjarAlias = 'U10';
```

Comment:
This is a bit ugly but the Hotjar tags are really strictly limited character-wise (25 or something) so I thought keeping the length the same was easiest.